### PR TITLE
drivers: remove RoboClaw

### PR DESCRIPTION
**Pull request by sfuhrer**
Is this driver something that's being used and we consider worth to maintain in PX4 core? 
It would need maintainance work to get https://github.com/PX4/PX4-Autopilot/pull/20837 in, plus removing it also reduces flash on v5 by 0.21%.

### CODE REVIEW

**Comment by MaEtUgR** [comment]
I spent some time with this driver before and had some suggestions to make one of our rovers go in a certain configuration. But never made a pr since it was deprioritized. If we remove the driver we can't use that rover anymore but we can also revisit it once it gets important again.

**Comment by dagar** [comment]
I have one of these, but using it isn't exactly high on my todo list.

If it's actively blocking https://github.com/PX4/PX4-Autopilot/pull/20837 and can't be resolved in a few minutes then let's delete it for now and bring it back properly later (if desired).

You can delete WheelEncoders.msg along with it.

**Comment by sfuhrer** [comment]
>If it's actively blocking https://github.com/PX4/PX4-Autopilot/pull/20837 and can't be resolved in a few minutes then let's delete it for now and bring it back properly later (if desired).

It's for sure a bit more than a couple of minutes, especially as it would need to be tested again. Maybe rovers get more interesting once we have proper ROS integration and we can revisit then?

File: ROMFS/px4fmu_common/init.d/airframes/50003_aion_robotics_r1_rover [hunk 1]
```diff
@@ -54,14 +54,6 @@ param set-default CBRK_AIRSPD_CHK 162128
 # Differential drive acts like ackermann steering with a maximum turn angle of 180 degrees, or pi radians
 param set-default GND_MAX_ANG 3.1415
 
-param set-default RBCLW_BAUD 8
```
Comment (dagar) [inline ROMFS/px4fmu_common/init.d/airframes/50003_aion_robotics_r1_rover:57]: Is it even worth keeping this airframe?

**Comment by dagar** [comment]
> > If it's actively blocking #20837 and can't be resolved in a few minutes then let's delete it for now and bring it back properly later (if desired).
> 
> It's for sure a bit more than a couple of minutes, especially as it would need to be tested again. Maybe rovers get more interesting once we have proper ROS integration and we can revisit then?

Yes I think it could be quite a good fit integrated with Nav2 at the right level. https://navigation.ros.org/

**Comment by hamishwillee** [comment]
Will this change be in PX4 v1.14?

**Pull request merged**